### PR TITLE
Switch to Github Actions, bump python and django

### PR DESCRIPTION
**Pull request by Mojken**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,68 +3,74 @@
...
-]
+install_requires = ["six", "content-io >= 1.2.5", "simplejson >= 3.2.0"]
 
 tests_require = [
-    'coverage',
-    'Markdown <= 2.4.1',
-    'Pillow <= 3.4.2',
+    "coverage",
+    "Markdown <= 3.3.0",
+    "Pillow <= 3.4.2",
```
Comment (Swamii) [inline setup.py:11]: We should bump this. We use `8.4.0` on bolist. Maybe at least 6?

File: setup.py [hunk 2]
```diff
@@ -3,68 +3,74 @@
...
-        'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
-        'Topic :: Software Development :: Libraries :: Python Modules',
+        "Environment :: Web Environment",
+        "Natural Language :: English",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: BSD License",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 2",
+        "Programming Language :: Python :: 2.7",
```
Comment (Swamii) [inline setup.py:61]: We can drop these two lines

File: setup.py [hunk 3]
```diff
@@ -3,68 +3,74 @@
...
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
```
Comment (Swamii) [inline setup.py:64]: Drop as well and add the new versions

File: tox.ini [hunk 4]
```diff
@@ -4,36 +4,31 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27-django{ 15, 16, 17, 18, 19, 110, 111             },
-          py35-django{             18, 19, 110, 111, 20, 21, 22 },
-          py36-django{                          111, 20, 21, 22 },
-          py37-django{                          111, 20, 21, 22 },
-          py38-django{                          111, 20, 21, 22 }
+envlist = py36-django{ 20, 21, 22, 30, 31, 32 },
+          py37-django{ 20, 21, 22, 30, 31, 32 },
```
Comment (Swamii) [inline tox.ini:8]: Drop 20 and 21 as well (two years since EOL)

File: tox.ini [hunk 5]
```diff
@@ -4,36 +4,31 @@
...
-       django111: Django>=1.11,<1.12
        django20: Django>=2.0,<2.1
        django21: Django>=2.1,<2.2
        django22: Django>=2.2,<2.3
-       py26,py27: importlib
-                  unittest2
+       django30: Django>=3.0,<3.1
+       django31: Django>=3.1,<3.2
+       django32: Django>=3.2,<3.3
+       py26,py27: unittest2
```
Comment (Swamii) [inline tox.ini]: Drop

File: setup.py [hunk 6]
```diff
@@ -3,68 +3,74 @@
...
-        '    :target: https://travis-ci.org/5monkeys/djedi-cms\n'
-        '.. image:: https://coveralls.io/repos/5monkeys/djedi-cms/badge.svg?branch=master\n'
-        '    :target: https://coveralls.io/r/5monkeys/djedi-cms?branch=master\n\n'
-        '.. _documentation: https://djedi-cms.org/\n'
-        '.. _source: https://github.com/5monkeys/djedi-cms\n\n'
+        ".. image:: https://djedi-cms.org/_static/djedi-portrait.svg\n\n"
+        "- Read the documentation_\n"
+        "- Browse the source_\n\n"
+        ".. image:: https://travis-ci.org/5monkeys/djedi-cms.svg?branch=master\n"
+        "    :target: https://travis-ci.org/5monkeys/djedi-cms\n"
```
Comment (Swamii) [inline setup.py:28]: This link wont be relevant so let's drop it and add a proper github actions link later

**Review by Swamii — COMMENTED** [review]
Nice 🎉 couple things left

File: setup.py (hunk 1 above)
Comment (Mojken) [reply setup.py:11]: Why not the same?

Comment (Swamii) [reply setup.py:11]: Actually, I don't think we should require a specific version at all

**Pull request merged**